Splash and tournament image

MechanicKim opened the pull request:
Android
- Changed spalsh image(animation - delay(3 sec) repeat)
- To improve performance, use android native library
  I added a line on ‘build.gradle’ to add the library.

Common
- In tournament page, I added a ‘coming soon’ image and text.(Stacy’s req)

MechanicKim committed:
rnpm link

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

# Conflicts:
#	android/app/build.gradle
#	android/app/src/main/res/values/strings.xml
#	android/settings.gradle
#	ios/Yoda/Info.plist

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

# Conflicts:
#	ios/Yoda/Info.plist

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge remote-tracking branch 'origin/master' into splash_anim

# Conflicts:
#	ios/Bridgeme.xcodeproj/project.pbxproj

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Commit but don’t push

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Android
- Changed spalsh image(animation - delay(3 sec) repeat)
- To improve performance, use android native library
  I added a line on ‘build.gradle’ to add the library.

Common
- In tournament page, I added a ‘coming soon’ image and text.(Stacy’s req)

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
Merge branch 'master' into jason_fix_1

ifndefdeadmau5 commented on android/app/build.gradle:
```diff
@@ -144,6 +155,8 @@ dependencies {
     compile 'com.google.firebase:firebase-messaging:10.0.1'
     compile 'com.google.android.gms:play-services:10.0.1'
     compile 'com.android.support:multidex:1.0.0'
+
```
Did you add a line space here intentionally? or not...

MechanicKim replied:
No! this is not my intention :-)

ifndefdeadmau5 commented on src/components/Main.js:
```diff
@@ -124,6 +127,18 @@ const styles = StyleSheet.create({
     shadowOpacity: 0.5,
     shadowRadius: 3,
   },
+  comingView: {
```
How about change name to 'comingSoon...'? I couldn't understand the meaning of it at first

MechanicKim replied:
Okay! I'll change this name. Thanks!

MechanicKim committed:
1. I added execution permission to ‘gradlew’ to generate signed app.
2. Fixed style errors in ‘Main.js’

ifndefdeadmau5 commented:
Awesome! LGTM with few comment. Thanks

[![Approved with PullApprove](https://img.shields.io/badge/code--review-approved-brightgreen.svg)](https://pullapprove.com/sv-bootcamp/bridgeme-client/pull-request/122/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=sv-bootcamp/bridgeme-client)

MechanicKim committed:
I reflected recommended opinion.

MechanicKim committed:
Merge branch 'master' of https://github.com/sv-bootcamp/yoda-client

MechanicKim committed:
I resolved conflicts in ‘Main.js’

Conflicts:
src/components/Main.js

I’ll check the style in this file later after editing it completely.

MechanicKim commented:
@ifndefdeadmau5
I resolved conflicts on my branch again! ^^

MechanicKim committed:
I restored project.pbxproj file
